### PR TITLE
Used latest official jre8-slim image for che master

### PR DESCRIPTION
**Pull request by skabashnyuk**
### What does this PR do?
Used the latest community-supported java8 JRE image for che server.
After we have this pr https://github.com/eclipse/che/pull/15201 merged nothing is preventing us to use jre instead of jdk.
Alpine image support has been discontinued in https://github.com/docker-library/openjdk/commit/3eb0351b208d739fac35345c85e3c6237c2114ec#diff-17b0a72d5a10e24142544a9dbc8effcb

### What issues does this PR fix or reference?
n/a

#### Release Notes
n/a

#### Docs PR
n/a

### CODE REVIEW

**Review by sleshchenko — APPROVED** [review]
I can't say that I really understand all the options we could go with here

But I'm OK with the proposed way and I don't have any objections :+1:

**Comment by skabashnyuk** [comment]
>I can't say that I really understand all the options we could go with here

- latest patches from the official repository for JVM and for operation system by itself.
- latest options for JVM. There are a lot of efforts to backport container-related features from new versions.
-  We don't need jdk components in runtime.

**Review by benoitf — CHANGES_REQUESTED** [review]

File: dockerfiles/che/Dockerfile [hunk 1]
```diff
@@ -5,35 +5,11 @@
...
-    CHE_IN_CONTAINER=true
-
-RUN echo "http://dl-4.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories && \
-    apk add --update curl openssl sudo bash && \
-    curl -sSL "https://${DOCKER_BUCKET}/builds/Linux/x86_64/docker-${DOCKER_VERSION}" -o /usr/bin/docker && \
-    chmod +x /usr/bin/docker && \
-    echo "%root ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers && \
-    rm -rf /tmp/* /var/cache/apk/*
-
+FROM openjdk:8-jre-slim
```
[the remark this answers is not in the document]
Comment (benoitf) [reply dockerfiles/che/Dockerfile]: I forgot to hit send last week so sorry for late notification

**Comment by skabashnyuk** [comment]
@benoitf your comments have been addressed.

**Review by benoitf — APPROVED** [review]

**Comment by skabashnyuk** [comment]
ci-test

**Pull request merged**